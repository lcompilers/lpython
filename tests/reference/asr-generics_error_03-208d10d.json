{
    "basename": "asr-generics_error_03-208d10d",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/generics_error_03.py",
    "infile_hash": "d09e6306de96ad3575262220417ea0af090d29b26c6fceb01e73fd6d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_error_03-208d10d.stderr",
    "stderr_hash": "e4803340775b153d9eafeae36cff39a9469260a0e139f0fc41e9317d",
    "returncode": 2
}