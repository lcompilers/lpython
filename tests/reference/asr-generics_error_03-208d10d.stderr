semantic error: Restriction mismatch with provided arguments
  --> tests/errors/generics_error_03.py:20:1 - 21:16
   |
20 |    def add_string(x: i32, y: i32) -> i32:
   |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
   |
21 |        return x + y
   | ...^^^^^^^^^^^^^^^^ 
